README.md

```diff
@@ -102,133 +244,97 @@ Please follow [OpenAPI Specification](#openapi-specification) to learn more
 ```
 
-If you want to avoid having to write the image repository every time, you can
-do so by modifying the provided `Makefile` in the project's root directory:
-replace `IMG ?= <repository>` with
-`IMG ?= example.com/your_name/image_name:tag_name` on the top.
-You can even just name it as `IMG ?= image_name:tag_name` if you later want to
-push it to your DockerHub account or if you just plan to use it locally.
-
-Please refrain from building the container with docker commands directly, i.e.
-`docker build . -t name:tag` as the provided method will
-also test the program before building it.
+Please read along to learn usage specific to Docker.
 
-Run the docker image with:
+#### Mount Service Account
 
-```bash
-docker run example.com/your_name/image_name:tag_name
-```
+Providing the service account is exactly the same as specified in
+[Service Directory](#service-directory), but in order to use the
+`--service-account` flag you need to first mount the file in the container with
+`-v` **before** any other flag.
 
-Please follow along for a usage example with docker and to learn how to
-provide a valid credentials file.
+With `-v` you first specify where the file is stored in your computer. Then,
+after a `:`, you specify where you wish to mount that file in the container,
+which is going to be the argument that `--service-account` will take.
 
-### Mount Credentials
-
-In order to work properly, CNWAN Reader needs a valid credentials file, i.e. a
-Google Cloud Service Account.  
-To learn more about Google Cloud Service Accounts, please visit [this page](https://cloud.google.com/iam/docs/service-accounts)
-or read [this guide](https://cloud.google.com/iam/docs/creating-managing-service-accounts).
-
-Supposing your service account is stored in `Desktop/cnwan-credentials` and
-is called `serv-acc.json`, use this command to mount the file under
-`/credentials` with name `serv-acc.json` in the docker image:
+For example: supposing the path to the service account on your computer is
+`~/Desktop/cnwan-credentials/serv-acc.json` and you want to mount it in the
+container as `/credentials/serv-acc.json`, the flag looks like this:
 
 ```bash
-docker run \
 -v ~/Desktop/cnwan-credentials/serv-acc.json:/credentials/serv-acc.json \
-my-image \
-servicedirectory \
---project my-project \
---region us-west2 \
---service-account ./credentials/serv-acc.json
 ```
 
-As you can see, the path to the credentials file in the `--service-account`
-flag must match the one where the file is mounted in the docker image, i.e.
-the part after the semicolon `:` in `-v`.  
-Please remember that the `-v` flag must come *before* the name of the
-image.
+Now you can use all other flags as specified in [Usage](#usage) and,
+specifically, you can use `--service-account` as
+`--service-account /credentials/serv-acc.json`.
 
-All other arguments are the same as described in [Usage](#usage).
+Read the next section for a full Docker example.
 
 ### Docker Example
```

Comment:
```suggestion
#### Docker Example
```

Reply:
I have amended and re-pushed this. Thank you for spotting it!